src/client/components/Profile/Profile.styling.css

```diff
@@ -85,18 +95,31 @@
   padding: 10% 20%;
 }
 
-.name-edit-btn, .phone-info, .email-info {
+.name-edit-btn,
+.phone-info,
+.email-info {
   align-items: baseline;
 }
 
-.user-name input, .phone-info input {
+.user-name input,
+.phone-info input {
   padding-right: 2%;
   line-height: 1px;
+  border-color: gray;
+}
+
+.profile-sign-out-button {
+  display: flex;
+  justify-content: flex-end;
+}
+
+button {
+  background-color: #007677;
 }
 
 .name-image-container p {
   font-size: 20px;
-  margin: 2% 2% 5%
+  margin: 2% 2% 5%;
```

Comment:
Although they work, we prefer working with `rem` units or `px` instead. Something to think about.

Reply:
These CSS props were given by Victoria and in global styling. Her PR is already merged into develop. Please confirm if I can edit them here in the 'profile.CSS' file.

Reply:
I don't mind, this was more for you to think about in the future.